src/components/Tiles/Tile.js

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-
+import { misc } from 'js/i18n/definitions';
 import { tilePropTypes } from './proptypes';
 
-const Tile = ({ url, text, compact }) => (
-  <Link className={compact ? 'coa-Tile--compact' : 'coa-Tile'} to={url}>
-    <div className="coa-Tile__content">
-      <p className="coa-Tile__text">{text}</p>
-      <i className="material-icons">arrow_forward</i>
-    </div>
-  </Link>
-);
+import guideIcon from 'images/guide_icon.png';
+
+const Tile = ({ url, text, compact, __typename }) => {
+  console.log("misc :", misc)
```

Comment:
just a note to clean up console.logs

Reply:
my silly logs always sneakin' by me... It's been rmoved